sovtokenfees/sovtokenfees/test/auth_map/test_auth_set_fees.py

```diff
@@ -81,4 +87,16 @@ def test_set_fees(helpers,
                                          old_value='*',
                                          new_value='*',
                                          constraint=sovtokenfees_auth_map[edit_fees.get_action_id()].as_dict)
+    resp = sdk_get_auth_rule_request(looper,
```

Comment:
I think it makes sense to rather update tests from ` sovtokenfees/sovtokenfees/test/test_get_auth_rule.py` (which are almost copy-paste from Node),

Reply:
Moved this check in test_get_auth_rule for sovtokenfees, but didn't move it in sovtoken because there are not test_get_auth_rule.